gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/editor/GravatarQuickEditorPage.kt

```diff
@@ -160,6 +170,8 @@ private fun NavGraphBuilder.addAvatarPickerGraph(
                 navArgument("email") { type = NavType.StringType },
                 navArgument("avatarId") { type = NavType.StringType },
             ),
+            exitTransition = { fadeOut() },
+            enterTransition = { fadeIn() },
```

Comment:
An alternative could be  (The video in the other comment uses these)
```
exitTransition = { slideOutOfContainer(AnimatedContentTransitionScope.SlideDirection.End) },
enterTransition = { slideIntoContainer(AnimatedContentTransitionScope.SlideDirection.Start) },
```

This one makes it more like the AltText pushed the picker out or appears on top of it. I don't have a strong preference.

Reply:
I've applied your suggestions! Thanks!